ptx/sec_deriv_basic_rules.ptx

```diff
@@ -423,7 +433,14 @@
           <figure xml:id="fig_sintangapprox">
             <caption>A graph of <m>f(x) = \sin(x)+2x+1</m> along with its tangent line approximation at <m>x=\pi</m></caption>
             <image xml:id="img_sintangapprox">
-              <description></description>
+              <shortdescription>
+                Graph of a sine function that is increased by 2.
```

Comment:
I think "increased by 2" is unclear. (Increased in what way?) 
How about: "The graph of f(x) is a wave that follows the line y=2x+1"